{"name": "test_authentication_error_anthropic", "status": "passed", "description": "Test Anthropic authentication error produces standard format.", "attachments": [{"name": "log", "source": "3d0b8b00-474d-4e0a-b7e8-f828e60da761-attachment.txt", "type": "text/plain"}], "start": 1757656801688, "stop": 1757656801688, "uuid": "ea773571-2cfa-4751-a50c-e2443caceefe", "historyId": "4cf8759afeb15f6d713e8628096554ed", "testCaseId": "4cf8759afeb15f6d713e8628096554ed", "fullName": "tests.unit.test_standardized_llm_errors.TestStandardizedErrors#test_authentication_error_anthropic", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "parentSuite", "value": "tests.unit"}, {"name": "suite", "value": "test_standardized_llm_errors"}, {"name": "subSuite", "value": "TestStandardizedErrors"}, {"name": "host", "value": "Joshuas-MacBook-Pro-2.local"}, {"name": "thread", "value": "36177-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.test_standardized_llm_errors"}], "titlePath": ["tests", "unit", "test_standardized_llm_errors.py", "TestStandardizedErrors"]}